README.md

```diff
@@ -41,7 +41,7 @@ yarn add @albert-team/rebloom
 const { BloomFilter } = require('@albert-team/rebloom')
 
 const main = async () => {
-  const filter = new BloomFilter('filtername')
+  const filter = new BloomFilter('filtername',{host:'10.66.254.46',port:'6379',password:'myPassword'})
```

Comment:
Hope this doesn't contain your real Redis server password. If you need an example with custom host & port, something like this should be sufficient:
```js
 const filter = new BloomFilter('filtername', { host: 'localhost', port: 6379 })
```

Furthermore, BloomFilter doesn't support `password` option here. Please recheck the API docs.

Reply:
Got it. I'll create a new PR with 
const filter = new BloomFilter('filtername',{host:'localhost', port:6379, redisClientOptions:{password: 'myPassword'}})
I suppose to add more connection details in readme.md, so that user can use rebloom easier